codec-http/src/main/java/io/netty/handler/codec/http/CookieUtil.java

```diff
@@ -16,9 +16,15 @@
 package io.netty.handler.codec.http;
 
 
+import java.util.BitSet;
+
 import io.netty.util.internal.InternalThreadLocalMap;
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+final class CookieUtil {
 
-final class CookieEncoderUtil {
+    private static final InternalLogger LOGGER = InternalLoggerFactory.getInstance(CookieUtil.class);
```

Comment:
Well, it's probably better put this way - all our code use `logger` instead of `LOGGER` for some reason (logger is stateful so it's not a constant) which might or might not make sense to you. Let's just use `logger`.